.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 repos:
   - repo: https://github.com/golangci/golangci-lint
-    rev: v1.54.2
+    rev: v1.56.2
```

Comment:
Do we have the option to use a less specific version spec?  I.e., how about `v1.56` instead of `v1.56.2`?